assets/app/styles/_components.less

```diff
@@ -229,18 +228,43 @@
       border: 1px solid transparent;
       padding: 10px;
       position: relative;
-      margin: 0 10px 10px 0;
+      margin-bottom: 10px;
       text-align: center;
       -webkit-font-smoothing: antialiased;
+
+      + .pod {
+        margin-left: 10px;
+      }
+
       .pod-status-label {
         &:before, &:after {
           font-family: 'PatternFlyIcons-webfont';
           font-weight: normal;
         }
       }
+      &.pod-multiple {
+        position: relative;
+        
+        .fa-ellipsis-h {
+          position: absolute;
+          right: 5px;
+          bottom: 0;
+          font-size: 14px;
+          color: #333;
+        }
+      }
       &.pod-pending {
         background-color: #ececec;
         .striped(@color: rgba(255,255,255,.45); @angle: 45deg);
+
+        &.pod-multiple {
+          box-shadow: 
+            1px 1px 0 rgba(0, 0, 0, 0.2),
+            3px 3px 0 #f4f4f4,
+            4px 4px 0 rgba(0, 0, 0, 0.2),
+            6px 6px 0 #f4f4f4,
+            7px 7px 0 rgba(0, 0, 0, 0.2);
```

Comment:
Easy to do for the three that are the same except for one color.  I think I'll leave the pod-failed as is since it is significantly different than the other three.